apps/condo/domains/marketplace/schema/MarketItemFile.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Generated by `createschema marketplace.MarketItemFile 'marketItem:Relationship:MarketItem:CASCADE; file:File;'`
+ */
+
+const get = require('lodash/get')
+
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+
+const FileAdapter = require('@condo/domains/common/utils/fileAdapter')
+const { getFileMetaAfterChange } = require('@condo/domains/common/utils/fileAdapter')
+const access = require('@condo/domains/marketplace/access/MarketItemFile')
+const { addOrganizationFieldPlugin } = require('@condo/domains/organization/schema/plugins/addOrganizationFieldPlugin')
+
+
+const MARKET_ITEM_FILE_FOLDER_NAME = 'marketitemfile'
+const Adapter = new FileAdapter(MARKET_ITEM_FILE_FOLDER_NAME)
+const fileMetaAfterChange = getFileMetaAfterChange(Adapter)
+const ALLOWED_MIME_TYPES = [
+    'image/jpeg',
+    'image/png',
+    'image/heic',
+]
+const ERRORS = {
+    NON_IMAGE_FILE: {
+        code: BAD_USER_INPUT,
+        type: 'NON_IMAGE_FILE',
+    },
+}
+
+const MarketItemFile = new GQLListSchema('MarketItemFile', {
+    schemaDoc: 'File attached to the market item',
```

Comment:
What is this file?
Will it be only for the image of the product or could it be some additional files (such as instructions)?